.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["es2015", "stage-0", "react"],
+	"presets": [
+    ["env", {"targets": { "electron": 1.4 }, "useBuiltIns": true }],
```

Comment:
@jhen0409 makes sense. At the moment, I believe node 6 and 7 have the exact same es6 compatibility.